src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -29,7 +29,7 @@ export class ButtonColumn<T extends Slick.SlickData> extends BaseClickableColumn
 	public get definition(): Slick.Column<T> {
 		return {
 			id: this.options.id || this.options.title || this.options.field,
-			width: this.options.showText === true ? this.options.width : 26,
+			width: this.options.showText === true ? this.options.width : 50,
```

Comment:
Why are you changing this? Typically we shouldn't be changing defaults like this since other users of this feature may be expecting it to be 26. Could you not just set the column width to 50 in your own code?